core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.ts

```diff
@@ -371,38 +386,40 @@ export class ContributionsAndReview
   getContributionSummaries(
       suggestionIdToSuggestions: Record<string, SuggestionDetails>
   ): ContributionsSummary[] {
-    if (this.activeSuggestionType === this.SUGGESTION_TYPE_TRANSLATE) {
+    if (this.activeTabSubtype === this.SUGGESTION_TYPE_TRANSLATE) {
       return this.getTranslationContributionsSummary(suggestionIdToSuggestions);
-    } else if (this.activeSuggestionType === this.SUGGESTION_TYPE_QUESTION) {
+    } else if (this.activeTabSubtype === this.SUGGESTION_TYPE_QUESTION) {
       return this.getQuestionContributionsSummary(suggestionIdToSuggestions);
     }
   }
 
-  getActiveDropdownTabChoice(): string {
-    if (this.activeTabType === this.TAB_TYPE_REVIEWS) {
-      if (this.activeSuggestionType === this.SUGGESTION_TYPE_QUESTION) {
-        return 'Review Questions';
-      }
-      return 'Review Translations';
-    }
+  getActiveDropdownTabChoice(tabType: string, subType: string): string {
```

Comment:
```suggestion
  getActiveDropdownTabText(tabType: string, subType: string): string {
```
"Active" already implies that the tab is chosen, and we're returning the tab text not the tab object itself.

Reply:
Done